.gitlab-ci/docker-ros.yml

```diff
@@ -231,6 +238,55 @@ Test run-arm64:
     _PLATFORM: arm64
 
 
+.slim:
+  stage: Slim Images
```

Comment:
Add `slim` as a `TARGET`?

Reply:
Would have to adjust logic of TARGET, so we decided to use `ENABLE_SLIM` flag instead.